tests/test_segmentation_islands_effect.py

```diff
@@ -0,0 +1,115 @@
+from pathlib import Path
+
+import numpy as np
+import pytest
+from undo_stack import UndoStack
+
+from trame_slicer.segmentation import SegmentationEffectIslands
+
+
+@pytest.fixture
+def undo_stack(a_segmentation_editor):
+    undo_stack = UndoStack()
+    a_segmentation_editor.set_undo_stack(undo_stack)
+    return undo_stack
+
+
+@pytest.fixture
+def a_segmentation_spheres_file_path(a_data_folder) -> Path:
+    return a_data_folder.joinpath("segmentation_spheres.nii.gz")
+
+
+@pytest.fixture
+def effect(a_segmentation_editor):
+    effect: SegmentationEffectIslands = a_segmentation_editor.set_active_effect_type(SegmentationEffectIslands)
+    return effect
+
+
+@pytest.fixture
+def segment_id(a_segmentation_editor):
+    return a_segmentation_editor.get_nth_segment_id(0)
+
+
+@pytest.fixture(autouse=True)
+def set_up(a_slicer_app, a_volume_node, a_segmentation_editor, a_segmentation_spheres_file_path):
+    a_slicer_app.display_manager.show_volume(a_volume_node, vr_preset="MR-Default")
+    segmentation_node = a_slicer_app.io_manager.load_segmentation(a_segmentation_spheres_file_path)
+    a_segmentation_editor.set_active_segmentation(segmentation_node, a_volume_node)
+
+
+def get_segment_array(
+    a_segmentation_editor,
+    segment_id,
```

Comment:
Auto format doesn't follow trame_slicer rules (remove suffix comma and run ruff / black)